pygeoapi/evaluate.py

```diff
@@ -0,0 +1,132 @@
+
+        # evaluation for Between Predicate Node
+        elif isinstance(node, BetweenPredicateNode):
+            return filters.between(
+                to_filter(node.lhs), to_filter(node.low), to_filter(node.high),
+                node.not_
+            )
+
+        # evaluation for Between Predicate Node
+        elif isinstance(node, BetweenPredicateNode):
+            return filters.between(
+                to_filter(node.lhs), to_filter(node.low), to_filter(node.high),
+                node.not_
+            )
+
+        # evaluation for Like Predicate Node
+        elif isinstance(node, LikePredicateNode):
+            return filters.like(
+                to_filter(node.lhs), to_filter(node.rhs), node.case, node.not_,
+                self.mapping_choices
+
+            )
+
+        # evaluation for In Predicate Node
+        elif isinstance(node, InPredicateNode):
+            return filters.contains(
+                to_filter(node.lhs), [
+                    to_filter(sub_node) for sub_node in node.sub_nodes
+                ], node.not_, self.mapping_choices
+            )
+
+        # evaluation for Null Predicate Node
+        elif isinstance(node, NullPredicateNode):
+            return filters.null(
+                to_filter(node.lhs), node.not_
+            )
+
+        # evaluation for Temporal Predicate Node
+        elif isinstance(node, TemporalPredicateNode):
+            return filters.temporal(
+                to_filter(node.lhs), node.rhs, node.op
+            )
+
+        # evaluation for Spatial Predicate Node
+        elif isinstance(node, SpatialPredicateNode):
+            return filters.spatial(
+                to_filter(node.lhs), to_filter(node.rhs), node.op,
+                to_filter(node.pattern),
+                to_filter(node.distance),
+                to_filter(node.units)
+            )
+        
+        # evaluation for BBox Predicate Node
+        elif isinstance(node, BBoxPredicateNode):
+            return filters.bbox(
+                to_filter(node.lhs),
+                to_filter(node.minx),
+                to_filter(node.miny),
+                to_filter(node.maxx),
+                to_filter(node.maxy),
+                to_filter(node.crs)
+            )
+
+        # evaluation for Attribute Expression Node
+        elif isinstance(node, AttributeExpression):
+            return filters.attribute(node.name, self.field_mapping)
+
+        # evaluation for Literal Expression Node
+        elif isinstance(node, LiteralExpression):
+            return node.value
+
+        # evaluation for Arithmetic Expression Node
+        elif isinstance(node, ArithmeticExpressionNode):
+            return filters.arithmetic(
+                to_filter(node.lhs), to_filter(node.rhs), node.op
+            )
+
+        # return the Node
+        return node
+
+
+def to_filter(ast, field_mapping=None, mapping_choices=None):
+    """ Helper function to translate ECQL AST to Django Query expressions.
```

Comment:
Refuse from Django integration?

Reply:
No, just missed to change this docstring. Fixed now.